deploy/mapping/kustomization.yaml

```diff
@@ -19,6 +19,4 @@ configMapGenerator:
   - name: schema-mapping
     files:
       - base.mcf
-      - dailyweather.mcf
```

Comment:
Same question as in https://github.com/datacommonsorg/website/pull/1500#issuecomment-1084994900 - should we wait until dailyweather lands in obs?